SConstruct

```diff
@@ -445,21 +437,12 @@ consolidateObjectTable = command("consolidateObjectTable", [transformObjectCatal
                                   validate(ConsolidateObjectValidation, DATADIR, patchDataId,
                                            gen3id=patchGen3id)])
 
-# TODO : Workaround for DM-22256. forcedPhotCoadd won't be needed.
 gen3repo = env.Command([os.path.join(REPO, "butler.yaml"), os.path.join(REPO, "gen3.sqlite3")],
                        [forcedPhotCcd, forcedPhotCoadd, consolidateObjectTable],
```

Comment:
Should be able to remove `forcedPhotCoadd` from this list here

Reply:
I actually had that change locally, but didn't push. Good catch. Thank you.